assignment3.cpp

```diff
@@ -0,0 +1,14 @@
+#include<iostream>
+using namespace std;
+
+int main(int argc, char* argv[]){
+  int i;
+  if(argc<=1){
+    cout<<"Enter any arguement";
+  }
+  else{
+    for(i=1;i<argc;i++){
```

Comment:
For loop can be modified to 
```
for(i=1;i<argc;i++)
    cout<<argv[i];
```

Reply:
@CTI120 I have added the demo comment for your reference purpose.

Reply:
Noted